ui/lib/core/addon/components/readonly-form-field.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @module ReadonlyFormField
+ * ReadonlyFormField components are used to...
+ *
+ * @example
+ * ```js
+ * <ReadonlyFormField @attr={attr} />
+ * ```
+ * @param {object} attr - Should be an attribute from a model exported with expandAttributeMeta
+ * @param {any} value - The value that should be displayed on the input
+ */
+
+import Component from '@glimmer/component';
+import { setComponentTemplate } from '@ember/component';
+import { capitalize } from 'vault/helpers/capitalize';
+import { humanize } from 'vault/helpers/humanize';
+import { dasherize } from 'vault/helpers/dasherize';
+import layout from '../templates/components/readonly-form-field';
+
+class ReadonlyFormField extends Component {
+  get labelString() {
+    if (!this.args.attr) {
+      return '';
+    }
+    const label = this.args.attr.options ? this.args.attr.options.label : '';
+    const name = this.args.attr.name;
+    if (label) {
+      return label;
+    }
+    if (name) {
+      return capitalize([humanize([dasherize([name])])]);
```

Comment:
Rather than having to have array wrappers to use the helpers (which are oriented toward templates), you could use the same `Ember.String` functions those helpers also use, like [`capitalize`](https://api.emberjs.com/ember/release/classes/String/methods/loc?anchor=capitalize). Slightly cleaner interface 😀